platformio.ini

```diff
@@ -56,8 +56,8 @@ lib_deps =
     adafruit/Adafruit BMP280 Library@^2.6.8
     pololu/VL53L0X@^1.3.1
     ; https://github.com/fhessel/esp32_https_server
-    https://github.com/amandel/esp32_https_server.git#hotfix/openbikesensor
-    ; TODO: https://github.com/jasenk2/esp32_https_server.git#esp_tls
+    ; https://github.com/amandel/esp32_https_server.git#hotfix/openbikesensor
+    https://github.com/jasenk2/esp32_https_server.git#esp_tls
```

Comment:
The fork does not include the CN fix - when I remember correctly this was mainly needed for Apple devices.

https://github.com/fhessel/esp32_https_server/commit/253013d9bfde6f0aaaef563065aee1c29f113f5d

Reply:
Thanks @schiermi  for the contribution!